### PR TITLE
Crm457 426/layout and styles

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
Add base styles and layout (copied from provider) to give a common base as we work forward

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRM457-426

## Reviewer note
This PR includes some extra items that we don't need (in the views) as it is copy and paste job.
I plan to tidy this us as we go, but as it is working for now and not too concerned about this.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+    pagination:
+      title: "Pagination navigation"
+      first_html: "<span class='govuk-pagination__link-title'>First <span class='govuk-visually-hidden'>page</span></span>"
+      last_html: "<span class='govuk-pagination__link-title'>Last <span class='govuk-visually-hidden'>page</span></span>"
+      previous_html: "<svg class='govuk-pagination__icon govuk-pagination__icon--prev' xmlns='http://www.w3.org/2000/svg' height='13' width='15' aria-hidden='true' focusable='false' viewBox='0 0 15 13'><path d='m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z'></path></svg><span class='govuk-pagination__link-title'>Previous <span class='govuk-visually-hidden'>page</span></span>"
+      next_html: "<span class='govuk-pagination__link-title'>Next <span class='govuk-visually-hidden'>page</span></span><svg class='govuk-pagination__icon govuk-pagination__icon--next' xmlns='http://www.w3.org/2000/svg' height='13' width='15' aria-hidden='true' focusable='false' viewBox='0 0 15 13'><path d='m8.107-0.0078125-1.4136 1.414 4.2926 4.293h-12.986v2h12.896l-4.1855 3.9766 1.377 1.4492 6.7441-6.4062-6.7246-6.7266z'></path></svg>"
```
Comment (pking-moj) [inline config/locales/en.yml:12]: Any reason we are storing html in the locales file?

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+      truncate: "&hellip;"
```
Comment (pking-moj) [inline config/locales/en.yml:13]: typo?

File: config/locales/en.yml (hunk 1 above)
Comment (dwhenry) [reply config/locales/en.yml:12]: no idea, I copied and pasted this from provider that come from crime apply.. not even 100% sure we use these ones... :( 

let me check and get back to you

File: config/locales/en.yml (hunk 2 above)
Comment (dwhenry) [reply config/locales/en.yml:13]: copied as above... let me check

Comment (dwhenry) [reply config/locales/en.yml:13]: also https://www.codecademy.com/forum_questions/4fb733f1d414ec0003003d2a

File: config/locales/en.yml (hunk 1 above)
Comment (dwhenry) [reply config/locales/en.yml]: removed - though will likely come back in later when we do pagination but worth a deeper review of the approach at that stage.

**Review by pking-moj — APPROVED** [review]

**Pull request merged**